src/src/DataStructs/Web_StreamingBuffer.cpp

```diff
@@ -260,23 +260,21 @@ void Web_StreamingBuffer::endStream() {
     if (buf.length() > 0) { sendContentBlocking(buf); }
     buf.clear();
     sendContentBlocking(buf);
-    #ifdef ESP8266
-    web_server.client().flush(100);
-    #endif
-    #ifdef ESP32
+
     web_server.client().flush();
-    #endif
+
     finalRam = ESP.getFreeHeap();
 
-    /*
+#ifndef BUILD_NO_DEBUG
         if (loglevelActiveFor(LOG_LEVEL_DEBUG)) {
         String log = String("Ram usage: Webserver only: ") + maxServerUsage +
                     " including Core: " + maxCoreUsage +
                     " flashStringCalls: " + flashStringCalls +
                     " flashStringData: " + flashStringData;
         addLog(LOG_LEVEL_DEBUG, log);
         }
-      */
+#endif // ifndef BUILD_NO_DEBUG
```

Comment:
The old state was that it was not even included in the 'normal' or 'max' builds, now you put it back in?

Reply:
Yes, like with all other debug outputs so it can be enabled or disabled with a define at compile time, depending if you need the size or the debug output. I think it's done everywhere like this?

Reply:
Point was, this was not needed in any build right now, as it is only for debugging purposes and if ever needed again a new build to enable it is perfectly fine :)

Reply:
Which is true for all DEBUG output I guess ;) But I'm also perfectly fine without it 😁 I just stumbled over it and included it because I thought it's always done with the ifdef's instead of directly uncommenting it...